package.json

```diff
@@ -32,7 +32,7 @@
     "test": "npm run test:mocha"
   },
   "engines": {
-    "node": ">=10.17.0"
+    "node": ">=14.13.1"
```

Comment:
We can set it to 14 as it is working flawless with 14.0.0
Do you agree?
```suggestion
    "node": ">=14"
```

Reply:
Sounds good to me 👍